frontend/src/shared/components/Header.tsx

```diff
@@ -1,17 +1,34 @@
-import { Center, Flex, Heading, Spacer } from '@chakra-ui/react';
+import { Button, Center, Flex, Heading, Spacer } from '@chakra-ui/react';
 import { Link } from 'react-router-dom';
 import { reverse } from '@shared/helpers/routing';
+import { useJwt } from '@users/contexts';
 
 export const Header: React.FC = () => {
+  const { jwt, removeJwt } = useJwt();
```

Comment:
Lipa ze header wie o tym, ze uzywamy jwt. To ten sam case, co kiedys byl z firestore. Z punktu widzenia headera nie powinno miec znaczenia, w jaki sposob user jest zalogowany. Moim zdaniem tu powinien byc uzyte cos w tym stylu

```
const { isLoggedIn, logout } = useUser();
```